waiter/src/waiter/config.clj

```diff
@@ -36,3 +38,11 @@
   []
   {:pre [(realized? config-promise)]}
   (get-in @config-promise [:waiter-principal]))
+
+(defn retrieve-exclusive-promotion-start-epoch-time
+  "Retrieves the configured (or system default) exclusive promotion start epoch time."
+  []
+  {:pre [(realized? config-promise)]}
+  (let [default-exclusive-promotion-start-time "2022-02-01T00:00:00.000Z"
```

Comment:
This date seems oddly specific for the open-source default... Can we have the default be "never" instead? Maybe set it to the year 3000?

Reply:
CHanged to `2050-06-01T00:00:00.000Z`